components/window/position.md

```diff
@@ -12,42 +12,14 @@ position: 2
 
 The Window offers several ways to control its position:
 
-* [`Centered` parameter](#centered)
 * [`ContainmentSelector` parameter](#containmentselector)
 * [`Top` and `Left` parameters](#top-and-left)
+* [Centered Window](#center)
+* [Positions Example](#example)
 
-The Window renders [in the root of the application](slug:window-overview#important-notes) or in its [containment element](#containmentselector). If the app is using special CSS positioning, margins, or other offsets on the Window ancestors, these CSS styles may [affect the position of the Window](slug:troubleshooting-general-issues#wrong-popup-position).
-
-
-## Centered
-
-The `Centered` parameter determines if the Window displays centered in the viewport. The parameter value is `true` by default.
-
-A centered Window applies the following CSS styles, which maintain the centered position even if the viewport size changes:
-
-````CSS.skip-repl
-.k-centered {
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-}
-````
-
-If the `Top` or `Left` parameters are set and not empty, they take precedence over `Centered`. To center the Window dynamically through its `Centered` parameter, bind the `Top` and `Left` parameters too, so you can reset them to `string.Empty` or `null`.
-
->caption Center the Window
-
-````RAZOR
-<TelerikWindow Centered="true" Visible="true">
-    <WindowTitle>
-        Window Title
-    </WindowTitle>
-    <WindowContent>
-        A Centered Window
-    </WindowContent>
-</TelerikWindow>
-````
+The Window renders [in the root of the application](slug:window-overview#important-notes) or in its [containment element](#containmentselector). However, if the `Top` and `Left` parameters are not defined or equal to `string.Empty`, the Telerik Window will be positioned at the center of the viewport by default. 
 
+If the application is using special CSS positioning, margins, or other offsets on the Window ancestors, these CSS styles may [affect the position of the Window](slug:troubleshooting-general-issues#wrong-popup-position). 
```

Comment:
The first and second sentence are linked with "however", but they are not logically related. Actually, the first and third sentence are more related and can be put together.

The second sentence assumes knowledge about parameters, which are not explained yet. Since it is repeated below and there is a dedicated heading, you can remove it from here.